src/main/java/com/epam/aidial/core/controller/DeploymentPostController.java

```diff
@@ -163,12 +163,14 @@ private void handleProxyRequest(HttpClientRequest proxyRequest) {
         if (!deployment.isForwardApiKey()) {
```

Comment:
Why do you change the original request?  Is it better to always add api-key header to excluded headers to not copy it and do not modify the original request.

Reply:
fixed

Reply:
fixed